include/morphio/morphology.h

```diff
@@ -36,8 +37,9 @@ class Morphology
         Example:
             Morphology("neuron.asc", TWO_POINTS_SECTIONS | SOMA_SPHERE);
      */
-    explicit Morphology(const std::string& source, unsigned int options = NO_MODIFIER);
-    explicit Morphology(mut::Morphology);
+    Morphology(const std::string& source, unsigned int options = NO_MODIFIER);
+    Morphology(const HighFive::Group& group, unsigned int options = NO_MODIFIER);
+    Morphology(mut::Morphology);
```

Comment:
@st4rl3ss I think all these should still be explicit?

Reply:
Thanks! Makes sense. Fixed it!